src/components/NewCommentForm.tsx

```diff
@@ -1,13 +1,12 @@
 import classNames from 'classnames';
 import React, { useState } from 'react';
-import { CommentData } from '../types/Comment';
+import { addComment } from '../features/postSlice';
+import { useAppDispatch, useAppSelector } from '../app/hooks';
 
-type Props = {
-  onSubmit: (data: CommentData) => Promise<void>;
-};
-
-export const NewCommentForm: React.FC<Props> = ({ onSubmit }) => {
-  const [submitting, setSubmitting] = useState(false);
+export const NewCommentForm = () => {
+  const submitting = useAppSelector(state => state.post.submitting);
+  const post = useAppSelector(state => state.post);
```

Comment:
```suggestion
  const { post } = useAppSelector(state => state.post);
```

Reply:
It's better to name it `postData` for example if you need to use the whole data object.